puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/testKatello.groovy

```diff
@@ -0,0 +1,170 @@
+def ruby = '2.4'
+
+pipeline {
+    agent { label 'fast' }
+
+    stages {
+        stage('Setup Git Repos') {
+            steps {
+                deleteDir()
+
+                sh "git config --global user.email 'ci@theforeman.org'"
+                sh "git config --global user.name 'jenkins'"
```

Comment:
Is stuff like these 2 lines better put in Puppet instead?

Reply:
I thought they were from my prior research, but they weren't working:

https://github.com/theforeman/foreman-infra/blob/7b7d83e2a75fa46879ab52806c30bc84cb27e909/puppet/modules/slave/files/gitconfig